addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -83,7 +83,7 @@ def done(self):
                 picking_to_backorder |= picking
             else:
                 picking.action_done()
-        if picking_without_qty_done:
+        if picking_without_qty_done == pickings:
```

Comment:
Not really a big deal, but `len(picking_without_qty_done) == len(pickings)` is slightly more performant.